astropy/table/column.py

```diff
@@ -648,6 +631,23 @@ def _copy_attrs(self, obj):
             setattr(self, attr, val)
         self.meta = deepcopy(getattr(obj, 'meta', {}))
 
+    @classmethod
```

Comment:
I think I'd make this a `staticmethod` since at present nothing is used that is stored on the class or instance. Then, if later we want, e.g., to use `self._encoding`, we can just change it to a `astropy.utils.sharedmethod` and no code would have to change.